src/components/data-management/NewProjectModal.jsx

```diff
@@ -44,6 +35,27 @@ const NewProjectModal = (props) => {
     existingNames: projectNames,
   };
 
+  useEffect(() => {
+    setProjectNames(new Set(projects.ids.map((id) => projects[id].name.trim())));
+  }, [projects.ids]);
+  useEffect(() => {
+    setIsValidName(validateInputs(projectName, validationChecks, validationParams).isValid);
+  }, [projectName, projectNames]);
+
+  const {
+    saving,
+    error,
+  } = useSelector((state) => state.projects.meta);
+
+  const submit = () => {
+    // TO-DO: this fails to clear the input field
+    // We prpbably can fix this taking advantage of
+    // the Form properties.
```

Comment:
Is this solved with `onFinish='submit'` on line 95?

Reply:
No, that's a failed attempt. I am going to remove it now